virttest/cpu.py

```diff
@@ -125,7 +129,7 @@ def get_cpu_xmldata(vm, options=""):
     cpu_xmldata['mtype'] = vm_xml.os.machine
     try:
         cpu_xmldata['current_vcpu'] = int(vm_xml.current_vcpu)
-    except LibvirtXMLNotFoundError:
+    except libvirt_xml.xcepts.LibvirtXMLNotFoundError:
```

Comment:
Just curious that as the `libvirt_xml.base` module has already been patched with the lazy importing fix of `virsh` do we really need `libvirt_xml.xcepts` to be imported in a lazy manner?

Reply:
I think I originally had `xcepts` lazily imported too but after the `libvirt_xml` was fixed here I unified both `libvirt_xml` and `libvirt_xml.xcepts` under the same lazy import. The current diff (at the time of speaking) has been integration tested and no libvirt warnings are emitted meaning that we don't need extra changes to make it work. Note however that some modules might have been imported this way simply because they contain `libvirt` functionality and as such might always become verbose too if somebody imports the virsh or other modules there. In that regard I considered taking this a step further from just tracing the warning and patching up the various locations it comes from right now and decided to isolate libvirt imports semantically for more future proof changes here.

Reply:
ok, understand your concern now, thanks!